docs/content/docs/deployment/memory/mem_setup_tm.md

```diff
@@ -98,13 +98,14 @@ See also [how to configure memory for state backends]({{< ref "docs/deployment/m
 If your job contains multiple types of managed memory consumers, you can also control how managed memory should be shared across these types.
 The configuration option [`taskmanager.memory.managed.consumer-weights`]({{< ref "docs/deployment/config" >}}#taskmanager-memory-managed-consumer-weights) allows you to set a weight for each type, to which Flink will reserve managed memory proportionally.
 Valid consumer types are:
-* `DATAPROC`: for RocksDB state backend in streaming and built-in algorithms in batch.
+* `OPERATOR`: for built-in algorithms in streaming or batch.
```

Comment:
Similar to `PYTHON`, we can omit "in streaming or batch"
```suggestion
* `OPERATOR`: for built-in algorithms.
```

Reply:
> Batch jobs can use it for sorting, hash tables, caching of intermediate results.

I think we should also update for this sentence, since now it applies to both streaming and batch.

(Github does not allow me to comment on lines that are not changed by this PR. :( )